zkevm-circuits/src/witness/tx.rs

```diff
@@ -299,12 +299,18 @@ impl Transaction {
                 Value::known(F::zero()),
                 rlc_be_bytes(&tx_hash_be_bytes, challenges.evm_word()),
             ],
+            [
+                Value::known(F::from(self.id as u64)),
+                Value::known(F::from(TxContextFieldTag::TxType as u64)),
+                Value::known(F::zero()),
+                Value::known(F::from(self.tx_type as u64)),
+            ],            
```

Comment:
This field should be placed after `BlockNumber` since the `TxType` tag is defined after block number.